Lib/test/support/script_helper.py

```diff
@@ -84,6 +85,21 @@ def fail(self, cmd_line):
                                 err))
 
 
+def run_until(condition, timeout=60, error_msg=None):
```

Comment:
Could this be documented at https://docs.python.org/3/library/test.html#module-test.support.script_helper ?